src/utils.js

```diff
@@ -213,6 +216,14 @@ export function isMultiSelect(schema) {
   return Array.isArray(schema.items.enum) && schema.uniqueItems;
 }
 
+export function isFixedItems(schema) {
+  return Array.isArray(schema.items) && schema.items.length > 0 && schema.items.every(item => isObject(item));
+}
+
+export function allowAdditionalItems(schema) {
+  return isObject(schema.additionalItems);
+}
```

Comment:
As spec says that ["additionalItems" MUST be either a boolean or an object](http://json-schema.org/latest/json-schema-validation.html#anchor37).

We should therefore fallback on the the default `items` schema if `allowAdditionalItems` its `true`.